app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -111,4 +112,25 @@ def create_network_router_queue(userid, options = {})
     }
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
+
+  def create_floating_ip(options)
```

Comment:
same as the previous PRs, please use the old format, the methods should not be here